README.md

```diff
@@ -7,9 +7,91 @@ Table of Contents
 =================
 
 * [Name](#name)
+* [Synopsis](#synopsis)
 * [Description](#description)
 * [Copyright & License](#copyright--license)
 
+Synopsis
+========
+
+Basic usage
+
+```
+
+# assignment in opsboy file scope.
+foo = bar; # Opsboy's assignment syntax. variable can be interpolated when startwith dollar sign and within strings.
+
+# comment in opsboy file.
+#foo = bar; # opsboy use # as comment syntax.
+
+# rule and target.
+
+Basically,
+
+    foo {
+	directive1;
+	directive2;
+    }
+
+is a rule. And foo is the target. It's quite similar to Makefile's rules perse.
+
+We use Opsboy's compile out perl and send make and target arguments to execute 
+it.
+
+./openresty-test.pl make foo 
+
+# command
+All opsboy command will list below.
+
+# git
+git git@github.com location;  #git clone resource to location.
+
+# file
+file location; # check if file exist, if not create.
+
+# running
+running 'process matching part'; # check process is running.
+
+# dep
+dep block1 block2 block3; # execute depend block.
```

Comment:
This comment is confusing. It should be

"specifying dependency rule names which must be checked and fulfilled before running the current rule"